helpers/compile/plugins/fill-plugin/fillers/events.ts

```diff
@@ -0,0 +1,36 @@
+type Listener = (...args: any[]) => void
+
+export class EventEmitter {
+  private events: Record<string, Listener[]> = {}
+
+  on(event: string, listener: Listener) {
+    if (!this.events[event]) {
+      this.events[event] = []
+    }
+
+    this.events[event].push(listener)
+
+    return this
+  }
```

Comment:
The current API only allows event listeners to be added.
Shouldn't event listeners also be removed somehow? Isn't there a risk of memory leaks?

Reply:
Nevermind, I see we never call `.off()` anyway.

Reply:
Prisma Client does not allow to remove listeners, so I don't think we need it until it is the case.